troubleshooting/app-does-not-connect-to-remote-point.md

```diff
@@ -0,0 +1,52 @@
+---
+title: App Cannot Connect to a Remote Service via HTTPS When Running on Device
+description: App cannot connect to a web service via HTTPS when deployed on a device and produces an unspecified error.
+page_title: App Cannot Connect to Remote Service via HTPS When Running on Device
+slug: app-cannot-connect to remote-service
+tags: connect, ios, web-service
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Run on Device</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+</table>
+
+## Description
+When running an app on a device, its requests to a remote point via HTTPS do not work, even though the app works in the device simulator. Upon expecting the error arguments of the failed request, you find `statusText: error` and no error message or response text.
+
+The issue is observed on iOS devices with any build configuration and on Android devices when the app is built with **Release** configuration.
+
+## Cause
+This problem occurs when the remote server uses an untrusted SSL certificate. This could happen if the server uses a certificate form a trusted authority but does not have any intermediate certificates installed, or if the server uses a self-signed certificate.
+
+Android devices allow connections via HTTPS to a server using an untrusted certificate only when you build the app with **Debug** build configuration, to let you test your app. When the app is built with **Release** configuration, connections to such servers are blocked.
+
+iOS devices block any HTTPS requests to untrusted remote points, no matter what build configuration was used to produce the app package.
+
+## Solution
+Use a trusted certificate on the server that you want to connect to. If you are already using a certificate issued by a trusted certificate authority, you need to check for any missing intermediate certificates that should link your certificate to the root certificate of the issuer. 
+
+## Suggested Workaround
+Sometimes, using a trusted certificate is not possible. For example, when connecting to a test server that you canot afford to buy a certificate for. In such scenarios, you can choose to modify the behavior of the iOS app and prevent it from blocking the request. You can achieve this using the third-party [Certificate Plugin](https://github.com/hypery2k/cordova-certificate-plugin). To do it follow these steps:
```

Comment:
the reason "cannot afford" is denigrating, better use something "you have not configured to use SSL"